blog/2015/10/more-about-jonah-duckles.html

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+root: ../../..
+author: Jonah Duckles
+title: "More About Jonah Duckles"
+date: 2015-10-06
+time: "12:00:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  I'm thrilled to be stepping into the role of Executive Director of
+  the Software Carpentry Foundation. I come to you as an experienced
+  instructional community member excited to share my administrative,
+  grant writing and professional backgrounds to help the Software
+  Carpentry Foundation become sustainable and reach exciting new
+  goals.
+</p>
+<!-- end excerpt -->
+<p>
+  I’ve taught Software Carpentry around the world and it has made me
+  passionate about spreading the movement. Scientists are
+  accumulating data faster than their training has prepared them and
+  they need new approaches to raise the the level of the technical
+  conversations when thinking about complicated research challenges.
+  Software Carpentry is an incredibly powerful approach for improving
+  scientific exploration in this data-driven era. It helps scientists
+  use their existing skills and do even more. Our work is important
+  and has impact!
+</p>
+<p>
```

Comment:
I suggest adding something along the lines of

"For those of you who don´t know me already, I come to you from a position as XX at the University of Oklahoma. 

You then continue with "the biggest success". I think this helps with the opening, and in many ways "makes it ok" for people not to know who you are. After all, we have >300 instructors, so :)